tests/ha/check_hawk.pm

```diff
@@ -24,13 +25,22 @@ sub run {
     barrier_wait("HAWK_INIT_$cluster_name");
 
     # Test the Hawk service
-    systemctl 'show -p ActiveState hawk.service | grep ActiveState=active';
-
-    # Test the Hawk port
-    assert_script_run "ss -nap | grep '.*LISTEN.*:$hawk_port\[[:blank:]]*'";
-
-    # Test Hawk connection
-    assert_script_run "nc -zv localhost $hawk_port";
+    if (systemctl 'show -p ActiveState hawk.service | grep ActiveState=active', ignore_failure => 1) {
+        # Test the Hawk port
+        assert_script_run "ss -nap | grep '.*LISTEN.*:$hawk_port\[[:blank:]]*'";
+
+        # Test Hawk connection
+        assert_script_run "nc -zv localhost $hawk_port";
+    }
+    else {
+        # Hawk is broken in SLE-15-SP1 we have an opened bug, so record it and continue in that case
+        if (is_sle('=15-sp1')) {
+            record_soft_failure 'Hawk is known to fail in 15-SP1 - bsc#1116209';
+        }
+        else {
+            record_info 'Hawk', 'Hawk is failing! Analysis is requiring and consider to open a bug if needed!';
```

Comment:
Shouldn't that be `die`?

Reply:
No because if we die we can't test the other part of HA. Hawk is not the "most" important part in our test and work is in progress to fully test Hawk in a separate test suite.

This is why I firstly used `test_flags` and let fatal to 0, to be sure that the test suite will continue even in case of Hawk failures (but it seems to not work anymore... I will open a poo# for this).